### PR TITLE
VACMS-14702: Add VBA related fields to service taxonomy

### PR DESCRIPTION
**Pull request by omahane**
## Description

Relates to #14702

## Testing done
Manually

### Term view
Screenshot 2023-08-29 at 8 50 26 AM

### Term edit
Screenshot 2023-08-29 at 8 50 39 AM

### [Add a term](https://pr15008-1h6x4utycqnbrxeok7xvwmmg96sanyxw.ci.cms.va.gov/admin/structure/taxonomy/manage/health_care_service_taxonomy/add)
- [x] Add a **Name**: [Test data] VBA Service
- [x] Add a **Health Service API ID**: testDataVbaService
- [x] In the VBA fieldset
  - [x] Check **Show for VBA Facilities**
  - [x] Click **Type of care** and pick "Veteran benefits" 
  - [x] Add a **VBA patient friendly name**: [Test data] patient friendly name of service
  - [x] Add **VBA common conditions**: [Test data] common conditions
  - [x] Add a **VBA description**: [Test data] VBA description
  - [x] Add an **Online self-service URL**: https://google.com
  - [x] Add an **Online self-service link text**: [Test data] Link text
  - [x] Save the term as published
  - [x] Confirm that the term saves and displays the content in the term view

### Create a service region
- Add a [Test data service region](https://va-gov-cms.ddev.site/node/add/service_region)
- Save as published

### Edit a term
Edit the "Test data" term you just created
- [ ] Add the test data service region to **VBA Service Regions**
- [ ] Save the term
- [ ] Confirm that the service region is showing on the term view

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `CMS Team`
- [ ] `Public websites`
- [x] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

### CODE REVIEW

**Comment by davidmpickett** [comment]
Type of care menu should be:
- Veteran benefits
- Family member & caregiver benefits
- Service member benefits
- Other services

**Comment by swirtSJW** [comment]
@omahane Looks good but I am not sure what to make of this discrepancy
image

**Comment by davidmpickett** [comment]
> @omahane Looks good but I am not sure what to make of this discrepancy image

I was not specific enough [when I told @omahane the source of truth](https://dsva.slack.com/archives/C0FQSS30V/p1693244689724179?thread_ts=1693244689.724179&cid=C0FQSS30V)  and he pulled the wrong info so I am being more specific now

**Comment by swirtSJW** [comment]
Oof.  Sorry the merge of my PR made some conflicts.

**Comment by omahane** [comment]
I must have gone to the wrong place in this document: 

> @omahane Looks good but I am not sure what to make of this discrepancy
> image

I had seen a different list from the document Dave shared.

File: config/sync/field.field.taxonomy_term.va_benefits_taxonomy.field_va_benefit_eligibility_ov.yml [hunk 1]
```diff
@@ -25,144 +25,144 @@ default_value_callback: ''
 settings:
   handler: 'default:paragraph'
   handler_settings:
-    negate: '0'
```
Comment (swirtSJW) [inline config/sync/field.field.taxonomy_term.va_benefits_taxonomy.field_va_benefit_eligibility_ov.yml:28]: Should this file be in this PR?   It seems unrelated to the work.

**Review by swirtSJW — APPROVED** [review]
image

**Comment by swirtSJW** [comment]
image
